README.rst

```diff
@@ -71,13 +71,81 @@ Getting started
 
 Are you interested in using ``scikit-package``? Begin with the ``Getting Started`` page in our online documentation at https://scikit-package.github.io/scikit-package!
 
+Installation
+------------
+
+The preferred method is to use `Miniconda Python
+<https://docs.conda.io/projects/miniconda/en/latest/miniconda-install.html>`_
+and install from the "conda-forge" channel of Conda packages.
+
+To add "conda-forge" to the conda channels, run the following in a terminal. ::
+
+        conda config --add channels conda-forge
+
+We want to install our packages in a suitable conda environment.
+The following creates and activates a new environment named ``skpkg_env`` ::
+
+        conda create -n skpkg_env scikit-package
+        conda activate skpkg_env
+
+To confirm that the installation was successful, type ::
+
+        python -c "import scikit_package; print(scikit_package.__version__)"
```

Comment:
because we have an app can we give instructions for getting the version from the app (`--version`), also instructions for running the help (`-h`) instead of this slightly clunky import above?

Since by default scikit package comes now with a minima, app, let's also change the template in the same way, though in that case let's leave also this line so it can be checked both ways.